packages/kbn-optimizer/limits.yml

```diff
@@ -57,7 +57,7 @@ pageLoadAssetSize:
   telemetry: 51957
   telemetryManagementSection: 38586
   transform: 41007
-  triggersActionsUi: 103400
+  triggersActionsUi: 104400
```

Comment:
This plugin is already above the limit and increasing it beyond this level is not something we can approve without explanation. Have you tried any of the debugging steps at https://www.elastic.co/guide/en/kibana/master/ci-metrics.html#ci-metric-resolving-overages?

All of the changes in this PR seem to be to the application itself which should never be loaded on page load, only when the application is actually being rendered.

Reply:
Ah, seeing https://github.com/elastic/kibana/pull/130710#issuecomment-1119843458 now

Reply:
Sounds good to me, thanks for the info

Reply:
FYI, in the future you could leave that as a comment on the limits.yml change itself and it'll be easier for us to spot 😉

Reply:
haha yea, thats my bad :sweat_smile: